robbery.js

```diff
@@ -44,7 +171,25 @@ exports.getAppropriateMoment = function (schedule, duration, workingHours) {
          * @returns {Boolean}
          */
         tryLater: function () {
-            return false;
+            if (allPossibleStarts.length === 0) {
+                return false;
+            }
+            var currentTime = new Date(startTime);
+            currentTime.setUTCMinutes(currentTime.getUTCMinutes() + COUNT_MINUTES_LATER);
+
+            return allPossibleStarts.some(function (period) {
+                var nextTime = period.from;
+                if (currentTime > nextTime) {
+                    nextTime = currentTime;
+                }
+                if ((period.to - nextTime) >= durationInMilliseconds) {
+                    startTime = nextTime;
+
+                    return true;
+                }
+
+                return false;
```

Comment:
Expected to return a value at the end of this function.at line 163 col 43 in robbery.js